kbase-extension/migrated

```diff
@@ -0,0 +1 @@
+2019-09-18T17:37:24.457793
```

Comment:
Nothing you did, but this file should probably just be deleted and added to `.gitignore`.